game_roll.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# game roll logic class
+class GameRoll
```

Comment:
what's the difference between this class and Frame ? methods are duplicated, maybe you add a base_frame with common methods then inherit from it in both classes